.github/workflows/sns_lifecycle.yml

```diff
@@ -0,0 +1,101 @@
+name: "SNS lifecycle"
+on:
+  push:
+jobs:
+  sns_lifecycle:
+    runs-on: ubuntu-latest
```

Comment:
We should probably opt for a deterministic pipeline, so I would fix the versions of all the dependencies.

Reply:
Pinned ubuntu to ubuntu-22.04.